ENIGMAsystem/SHELL/Platforms/SDL/Android/Window.cpp

```diff
@@ -0,0 +1,57 @@
+/** Copyright (C) 2019 Samuel Venable
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+**/
+
+#include "Platforms/General/PFwindow.h"
+#include "Platforms/SDL/Window.h"
+// #include <?????.h> idk what android-specific header(s) define(s) ANativeWindow * and EGLSurface...
+#include <SDL2/SDL.h>
+// #include <SDL2/SDL_syswm.h> // pussy not found !
+
+namespace enigma {
+
+// ANativeWindow *ANWin;
+// EGLSurface EGLSurf;
+
+// called from initGameWindow()
+// capture sdl window hwnd/surf
+void window_id_init() {
+  /*
```

Comment:
youshould mark stuff like this with TODO or FIXME so it's easier for people to find